.github/workflows/flatpak.yml

```diff
@@ -1,3 +1,5 @@
+---
+
 name: Flatpak (Experimental)
```

Comment:
Worth removing "Experimental" from this now?

Reply:
Sure!